packages/@o3r/pipeline/schematics/ng-add/templates/github/__dot__github/workflows/main.yml

```diff
@@ -0,0 +1,67 @@
+name: Main CI
+
+env:
+  DEFAULT_BRANCH: main
+  DEFAULT_BRANCH_PRERELEASE_NAME: prerelease
+  RELEASE_BRANCH_REGEX: 'release\/(0|[1-9]\d*)\.(0|[1-9]\d*)(\.0-(?:next|prerelease|rc))?$'
+
+on:
+  push:
+    branches:
+      - main
+      - 'release/*'
+  pull_request:
+    branches:
+      - main
+      - 'release/*'
+
+jobs:
+  build:
+    runs-on: <%= runner %>
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: ./.github/actions/setup
+      - name: Build
+        run: <%= packageManager %> run build
+
+  unit-tests:
+    runs-on: <%= runner %>
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: ./.github/actions/setup
+      - name: Unit tests
+        run: <%= packageManager %> run test
+
+  lint:
+    runs-on: <%= runner %>
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: ./.github/actions/setup
+      - name: Lint
+        run: <%= packageManager %> run lint
+
+  version:
+    permissions:
+      contents: write
+    runs-on: <%= runner %>
+    outputs:
+      nextVersionTag: ${{ steps.new-version.outputs.nextVersionTag }}
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: actions/setup-node@0a44ba7841725637a19e28fa30b79a866c81b0a6 # v4.0.4
```

Comment:
not sure this step is actually needed (sorry I said the opposite in a previous comment but I think it was a mistake)

Reply:
not needed indeed.